packages/e2e-test-utils/src/create-url.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Node dependencies
```

Comment:
> There was probably some reason to use `Node` keyword but I'm happy to see it merged into `External` group 👍

I suppose the idea is the imports are from the [Node.js API](https://nodejs.org/api/), not any explicit dependency. I could see there being some value in it, but it's not something we either codified or use consistently. Same goes for "Browser dependencies".

As part of this overall effort, I'm fine to make adjustments from what we've standardized. But the important thing is forcing ourselves to have these conversations, rather than let inconsistencies grow via the unwritten conventions of a few people.